packages/amplify_auth_cognito/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.3.0-rc.2 (2021-11-05)
```

Comment:
The date is wrong, but consistent with the other change logs. Probably make sense to have one PR update all of them if we want to update it

Reply:
Ahh yea that date is outdated. Agreed.